.github/workflows/ci.yml

```diff
@@ -23,6 +28,7 @@ jobs:
       - name: Install dependencies
         run: pnpm install --shamefully-hoist
       - name: Run Tests
+        if: matrix.os != 'windows-latest' && matrix.node-version != '20.18.0'
```

Comment:
tests won't run on Windows just for Node 20.18.0. They run for newer versions so I thought it wouldn't be a big deal. Fixing this would not be easy otherwise